conda-recipe/meta.yaml

```diff
@@ -24,6 +24,7 @@ requirements:
         - make  # [unix]
         - ninja  # [win]
         - numpy >=1.17
+        - gtest
```

Comment:
gtest 1.10 is needed as I am using GTEST_SKIP that only present in 1.10.

Reply:
```suggestion
        - gtest >=1.10
```